x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/exceptions/workflows/trial_license_complete_tier/role_based_rule_exceptions_workflows.ts

```diff
@@ -1,984 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Appears deleted, but just broke this file up into `prebuilt_rules.ts` and `rule_exceptions_execution.ts` and removed some tests that were redundant.